tendermint/src/lib.rs

```diff
@@ -18,7 +18,12 @@
     html_root_url = "https://docs.rs/tendermint/0.10.0"
 )]
 
+// NOTE(EB): can't figure out how to easily remove the extern crate per Rust2018 upgrade ...
+#[allow(unused_extern_crates)]
 extern crate prost_amino as prost;
```

Comment:
Probably the easiest way to actually fix this is through the Cargo.toml:

```toml
prost = { package = "prost-amino", version = "0.4.0" }
prost-derive = { package = "prost-amino-derive", version = "0.4.0" }
```

Reply:
I wasn't able to get this working. When I try it and then comment out these extern crate lines, I get tons of errors.

Reply:
@ebuchman oh, I bet it hasn't been updated to the 2018 edition, so `prost-amino-derive` is still dependent on the `extern crate` directives.

Probably the best solution would be to switch to upstream `prost` when Amino has been fully replaced with protos.

Reply:
lol cannot wait!